### PR TITLE
refactor: documentation for agent setup

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What?
- Refactored documentation for Agent setup section

### CODE REVIEW

File: src/content/docs/en/contributorsguide/setup/agent/basicsetup/npm.md [hunk 1]
```diff
@@ -3,3 +3,112 @@ title: Local setup using NPM
...
+  "outboundTransport": ["http"],
+  "adminPort": 4001,
+  "tenancy": false,
+  "webhookUrl": "http://localhost:5000/wh/9fb768a6-2ad8-435d-95f0-ad1876d2eb74",
+  "schemaFileServerURL": "https://schema.credebl.id/schemas/",
+  "didRegistryContractAddress": "0xcB80F37eDD2bE3570c6C9D5B0888614E04E1e49E",
+  "schemaManagerContractAddress": "0x4742d43C2dFCa5a1d4238240Afa8547Daf87Ee7a",
+  "rpcUrl": "https://rpc-amoy.polygon.technology",
+  "fileServerUrl": "https://schema.credebl.id",
+  "fileServerToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJBeWFuV29ya3MiLCJpZCI6ImNhZDI3ZjhjLTMyNWYtNDRmZC04ZmZkLWExNGNhZTY3NTMyMSJ9.I3IR7abjWbfStnxzn1BhxhV0OEzt1x3mULjDdUcgWHk"
```
Comment (pallavighule) [inline src/content/docs/en/contributorsguide/setup/agent/basicsetup/npm.md:72]: use dummy token here .

File: src/content/docs/en/contributorsguide/setup/agent/basicsetup/npm.md [hunk 2]
```diff
@@ -3,3 +3,112 @@ title: Local setup using NPM
...
+
+Below is the cliConfig.json file, which needs to be used to set up the agent
+```bash
+
+{
+  "label": "AFJ Rest Agent 1",
+  "walletId": "sample",  # Mention your walletId here
+  "walletKey": "sample",   # Mention your walletKey here
+  "walletType": "postgres",  # Mention wallet type here
+  "walletUrl": "localhost:5432",  # Mention your wallet type here
```
Comment (pallavighule) [inline src/content/docs/en/contributorsguide/setup/agent/basicsetup/npm.md:32]: check comment once.

Comment (bhavanakarwade) [reply src/content/docs/en/contributorsguide/setup/agent/basicsetup/npm.md:32]: Resolved

File: src/content/docs/en/contributorsguide/setup/agent/basicsetup/npm.md (hunk 1 above)
Comment (bhavanakarwade) [reply src/content/docs/en/contributorsguide/setup/agent/basicsetup/npm.md:72]: Resolved

**Pull request closed without merging**